src/Tools/onnx-mlir-opt/CMakeLists.txt

```diff
@@ -14,4 +14,5 @@ add_onnx_mlir_executable(onnx-mlir-opt
   MLIRLinalgTransforms
   MLIRMemRefTransforms
   MLIROptLib
+  MLIROpenMP
```

Comment:
This should not be needed here, since it is a library needed by CompilerOptions.